http_client_wrappers.go

```diff
@@ -0,0 +1,122 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to you under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package avatica
+
+import (
+	"fmt"
+	"net/http"
+
+	digest_auth_client "github.com/xinsnake/go-http-digest-auth-client"
+	"gopkg.in/jcmturner/gokrb5.v7/client"
+	"gopkg.in/jcmturner/gokrb5.v7/config"
+	"gopkg.in/jcmturner/gokrb5.v7/credentials"
+	"gopkg.in/jcmturner/gokrb5.v7/keytab"
+	gokrbSPNEGO "gopkg.in/jcmturner/gokrb5.v7/spnego"
+)
+
+// WrapWithDigestAuth takes an http client and prepares it to authenticate using digest authentication
+func WrapWithDigestAuth(cli *http.Client, username, password string) *http.Client {
```

Comment:
Rename this to `WithDigestAuth` as `Wrap` is redundant.

Reply:
done